pages/contributors.js

```diff
@@ -0,0 +1,113 @@
+import { React, useState, useEffect} from 'react'
+import Page from '../components/Page'
+
+export default function About() {
+    const [contributors, setContributors] = useState([])
+    
+    useEffect(() => {
+      const fetchContributors = async () => {
+        const response = await fetch(
+          'https://api.github.com/repos/carbon-app/carbon/contributors'
```

Comment:
I think you will want to increase the page size here, because right now it is only pulling the first 30

```
https://api.github.com/repos/carbon-app/carbon/contributors?per_page=100
```

We actually have over 100 contributors, though.

Reply:
Ahh got it! I don't think Github has set a limit for the "per page", so for now I just set it as 500, I will look into it more and see if there is a max/full option :)